docker/data-science/notebooks/ch14/2.question-answering-CPU-data-preparation.ipynb

```diff
@@ -74,24 +71,19 @@
   },
   {
    "cell_type": "code",
-   "execution_count": 8,
+   "execution_count": 43,
    "metadata": {},
    "outputs": [],
    "source": [
     "def get_context_dataframe(questions):\n",
     "    contexts = {\"id\": [], \"question\":[], \"context\":[], \"url\": []}\n",
     "    for question in questions:\n",
-    "        request = {\n",
-    "            \"query\": \"*:*\",\n",
-    "            \"fields\": [\"body\"],\n",
-    "            \"params\": {\n",
-    "              \"fq\": [\"id:\" + str(question[\"accepted_answer_id\"])],\n",
-    "              \"defType\": \"edismax\",\n",
-    "              \"rows\": 1,\n",
-    "              \"sort\": \"score desc\"\n",
-    "            }\n",
-    "        }\n",
-    "        docs = engine.docs_from_response(outdoors_collection.search(request))\n",
+    "        request = {\"query\": \"*:*\",\n",
```

Comment:
Throughout the code and book, let's please replace "*:*" with just "*" in our engine-agnostic queries. I think in Solr queries that * is also valid, so we likely don't need *:* anywhere.

Reply:
Done.